snippets/core/system-text-json/csharp/WeatherForecastCallbacksConverter.cs

```diff
@@ -11,14 +11,13 @@ public class WeatherForecastCallbacksConverter : JsonConverter<WeatherForecast>
             Type type,
             JsonSerializerOptions options)
         {
-            // Place "before" code here (OnDeserializing), but note that there is no access here to the POCO instance.
-            Console.WriteLine("OnDeserializing");
```

Comment:
Why were the `Console.WriteLines` removed?

Reply:
My comment here (https://github.com/dotnet/samples/pull/1837#discussion_r361065269) was specifically about `WeatherForecastRuntimeIgnoreConverter.cs` which didn't need these. It makes sense to have these in the `callback` sample.

Reply:
Got it, will restore them.